notifications_utils/markdown.py

```diff
@@ -201,6 +210,15 @@ def header(self, text, level, raw=None):
                     "-" * self.COLUMN_WIDTH,
                 )
             )
+        elif level == 2:
+            return "".join(
+                (
+                    self.linebreak() * 3,
+                    text,
+                    self.linebreak(),
+                    "-" * (self.COLUMN_WIDTH // 2),
```

Comment:
github allows `======` for h1s and `-------` for h2s, would it be worth us adopting that pattern in our plaintext?

this is an h1
========

this is an h2
-----------

```
this is an h1
========

this is an h2
-----------
```

Reply:
Will tweak 👍

Reply:
Ah, we already use `========` for hrules - is that still going to be fine?

Reply:
Provisionally added in 99a110aafb6db6c6b9417d19dc76cf511b7c37cb

Reply:
I can get on board with option 2 as well 👍

Reply:
Tweaked in 076f9744ad49d2fcd3a3cbf3783f343d76e22db6